amazon/ionbenchmark/Command.py

```diff
@@ -1,8 +0,0 @@
-from enum import Enum
-
-
-class Command(Enum):
```

Comment:
Accidentally deleted?

Reply:
Good catch.

It's such a simple PR, that I thought I could get away without checking myself before posting it. 😅
...but I was wrong.